### PR TITLE
adds gpu parameters

### PR DESCRIPTION
**Pull request by tkmamidi**
- [x] Adds GPU parameters to the startup script
  - default GPUs requested are zero

Use this command to test - 
`python3 ./start_vscode_tunnel.py -c 2 -g 1 -p amperenodes`

### CODE REVIEW

File: vscode_remote_tunnel/start_vscode_tunnel.py [hunk 1]
```diff
@@ -145,10 +147,18 @@ def main(args):
...
+        metavar="",
+    )
+
     PARSER.add_argument(
         "-m",
-        "--mem-per-cpu",
-        help="Memory per CPU to request",
+        "--mem",
+        help="Total memory to request",
         default="8G",
```
Comment (ManavalanG) [inline vscode_remote_tunnel/start_vscode_tunnel.py:162]: Now that we are switching to `--mem`, please bump up the default to `16G` as the script requests 2 cpus by default. `12G` may be acceptable as well.

File: vscode_remote_tunnel/start_vscode_tunnel.py [hunk 2]
```diff
@@ -34,22 +34,23 @@ def create_dirpath(arg):
...
+
     newline = "\n"  # newline to use inside fstring (https://stackoverflow.com/a/44780467/3998252)
     script_txt = f"""\
         #!/bin/bash
         #
         #SBATCH --job-name=vscode_tunnel
         #SBATCH --ntasks=1
         #SBATCH --cpus-per-task={cpu}
-        #SBATCH --mem={mem_per_cpu}
+        #SBATCH --gres=gpu:{gpu}
```
Comment (ManavalanG) [inline vscode_remote_tunnel/start_vscode_tunnel.py:52]: Any reason `--gres` was chosen over `--gpus`?

**Review by ManavalanG — CHANGES_REQUESTED** [review]

File: vscode_remote_tunnel/start_vscode_tunnel.py (hunk 2 above)
Comment (tkmamidi) [reply vscode_remote_tunnel/start_vscode_tunnel.py:52]: That is how the RC docs suggest for requesting GPUs

image

File: vscode_remote_tunnel/start_vscode_tunnel.py (hunk 1 above)
Comment (tkmamidi) [reply vscode_remote_tunnel/start_vscode_tunnel.py:162]: It was always requesting the total memory. I only changed the wording so it reflects what it was requesting. If 8G was working fine before, we can just leave it as is?

Comment (ManavalanG) [reply vscode_remote_tunnel/start_vscode_tunnel.py:162]: Good catch. Let's leave it as it is :)

**Review by ManavalanG — APPROVED** [review]
Good to go :)

**Pull request merged**